tests/test_stemma_dummy.py

```diff
@@ -0,0 +1,59 @@
+"""
+Unit tests for the StemmaDummy class.
+"""
+import unittest
+from stemmabench.algorithms.stemma_dummy import StemmaDummy
+
+
+class TestStemmaDummy(unittest.TestCase):
+    """Unit tests for the stemmaDummy class.
+    """
+    
+    def setUp(self) -> None:
+        """Setup the unit test.
+        """
+        self.stemma_folder_path = "tests/test_data/test_stemma"
+        self.stemmaDummy_empty = StemmaDummy()
+        self.stemmaDummy = StemmaDummy(self.stemma_folder_path, width=5)
+        self.stemmaDummy_seed = StemmaDummy(folder_path=self.stemma_folder_path, seed=1)
+        self.stemmaDummy.compute()
```

Comment:
this is what you want to test ;) 
should not be instantiated in the setup !

Reply:
Removed